pkg/k8shandler/reconciler.go

```diff
@@ -74,13 +73,13 @@ func ReconcileForClusterLogForwarder(forwarder *logging.ClusterLogForwarder, req
 
 	clusterLogging := clusterLoggingRequest.getClusterLogging()
 	if clusterLogging == nil {
-		logger.Debug("No clusterlogging object, cannot reconcile")
+		log.Info("No clusterlogging object, cannot reconcile")
 		return nil
 	}
 	clusterLoggingRequest.Cluster = clusterLogging
 
 	if clusterLogging.Spec.ManagementState == logging.ManagementStateUnmanaged {
-		logger.Debugf("Unmanaged state, nothing to reconcile")
+		log.Info("Unmanaged state, nothing to reconcile")
```

Comment:
same here

Reply:
this would be very noisy in the operator logs, i would recommend removing them

Reply:
deleted